.github/workflows/pr.yml

```diff
@@ -150,4 +150,23 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           check-unmerged-pr: true
+
+  check-docs:
+    name: Check documentation and README
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Check roles are documented in main README
+        run: |
+          echo "# Missing roles in README" > ${GITHUB_STEP_SUMMARY}
+          for role in $(find roles/ -maxdepth 1 -type d | sed -e 's,roles/,,'); do
+            if ! grep -q "^\[redhatci\.ocp\.${role}" README.md; then
```

Comment:
Ah! I see, thanks! This one I did it on purpose, because there is a particular directory (vendors) that is not a role, but it includes other roles.

:thinking: I need to figure out how to validate those sub-dirs with roles. I'll give it a second thought.

Reply:
No need, it was only a nit, lgtm